### PR TITLE
Login and Create Account

### PR DESCRIPTION
**Pull request by jude-lieb**
Initial html pages for logging in that include form verification.

### CODE REVIEW

File: newsApp/login/createAccount.html
Comment (zeshengchen) [inline newsApp/login/createAccount.html]: please move javascript to a separate file. moreover, consider to how to organize the project. For example, under root, there may have two folders: one for "frontend" and the other for "backend". For "frontend", there maybe has a folder to store "css" and the other to store "javascript" code.

File: newsApp/login/login.html
Comment (zeshengchen) [inline newsApp/login/login.html]: move javascript code into a different file.

**Review by zeshengchen — CHANGES_REQUESTED** [review]

**Review by zeshengchen — APPROVED** [review]

**Review by Pranaav003 — APPROVED** [review]

**Comment by zeshengchen** [comment]
@Pranaav003 for future pull requests, please wait until all group members approve it and then complete. For this pull request, @cgibson1421 has not approved it yet.

Moreover, please let who creates the pull request to complete it.

**Pull request merged**
